runtime/src/lib.rs

```diff
@@ -240,22 +240,28 @@ impl pallet_sudo::Config for Runtime {
 
 impl pallet_ethereum_chain_id::Config for Runtime {}
 
-pub const GAS_PER_SECOND: u128 = 4_000_000;
-pub const WEIGHT_PER_GAS: u128 = WEIGHT_PER_SECOND as u128 / GAS_PER_SECOND;
-pub struct GasMapper {}
+// Current (safe) approximation of the gas/s consumption considering
+// EVM execution over compiled WASM.
```

Comment:
```suggestion
/// Current (safe) approximation of the gas/s consumption considering
/// EVM execution over compiled WASM.
```

Let's make them doc comments?